torch/nn/quantized/_reference/modules/conv.py

```diff
@@ -0,0 +1,136 @@
+import torch
+import torch.nn.quantized as nnq
+import torch.nn.functional as F
+from typing import Optional
+from torch.nn.common_types import _size_1_t
+from torch.nn.modules.utils import _single
+
+class _ConvNd(nnq._ConvNd):
+    """ A reference version of nn.quantized.Conv2d
+        we will not pack the parameters in this module, since weight packing is an
+        optimization for quantized backends supported in PyTorch (fbgemm/qnnpack),
+        this is useful when user want to use this module in other backends like Glow.
+    """
+    __annotations__ = {"_bias": Optional[torch.Tensor]}
+
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super()._save_to_state_dict(destination, prefix, keep_vars)
+        destination[prefix + '_qweight'] = self._qweight
+        destination[prefix + '_bias'] = self._bias
+
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
```

Comment:
I didn't check what's currently in the corresponding functions in nnq._ConvNd, but is there a way to reuse these / etc, just to minimize the LOC footprint?  Makes sense if there is a good reason for it.

Reply:
they are not reusable as commented below.